packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_ExternalLibrary.res

```diff
@@ -18,9 +18,11 @@ let dispatch = (call: ExpressionValue.functionCall, environment, chain): result<
   expressionValue,
   'e,
 > =>
-  ReducerInterface_GenericDistribution.dispatch(call, environment) |> E.O.default(
-    chain(call, environment),
-  )
+  switch ReducerInterface_GenericDistribution.dispatch(call, environment) {
+  | Some(r) => r
+  | None =>
+    ReducerInterface_DateTime.dispatch(call, environment) |> E.O.default(chain(call, environment))
```

Comment:
I don't see just yet how this will work with lots of smaller specialized dispatch functions.

Reply:
It's easy to write functions to abstract this; we have one in this PR, but that isn't merged yet. 
https://github.com/quantified-uncertainty/squiggle/pull/545